src/modules/whats-new/components/whats-new-modal.tsx

```diff
@@ -57,6 +57,14 @@ const PageContent = ( {
 export default function WhatsNewModal( { showModal, onClose }: WhatsNewModalProps ) {
 	const locale = useI18nLocale();
 	const whatsNewPages: WhatsNewPage[] = [
+		{
+			image: selectiveSyncIllustration,
+			title: __( 'Choose what to sync' ),
+			description: __(
+				'Selective Sync allows you to sync only the files and folders you need, reducing the time and space required for your syncs.'
+			),
+			learnMoreUrl: getLocalizedLink( locale, 'docsSync' ),
```

Comment:
This learn mode url needs to be updated as soon as we have the docs added/updated.

Reply:
Let's link to the pull section where we'll update the screenshot and explain more about the selective sync.

```suggestion
			learnMoreUrl: getLocalizedLink( locale, 'docsSync' ) + '#pull',
```